### PR TITLE
Cockpit 288

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: _posts/2023-03-22-cockpit-288.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+The PCP metrics graphs have been redesigned so as to show by default more data in a single page view. 
+ 
+![screenshot of new design with more info in default view]({{ site.baseurl }}/images/288-metrics-new-design-with-more-info-in-default-view.png)
+![screenshot of new design with more info in default view]({{ site.baseurl }}/images/288-metrics-new-design-with-more-info-in-default-view-2.png)
+
+## API: Drop cockpit.dbus.publish() and .meta()
+
+The `cockpit.js` library dropped these two functions, which were meant to create a D-Bus server object from inside a Cockpit page. No [known Cockpit project](https://cockpit-project.org/applications.html) has ever used this, and we are not aware of a good use case. If you are using this API, please [let us know](https://github.com/cockpit-project/cockpit/issues)!
+
+## Accounts: Show shell and home directory
```
Comment (KKoukiou) [inline _posts/2023-03-22-cockpit-288.md:28]: `Show user's shell and home directory` maybe?

File: _posts/2023-03-22-cockpit-288.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+The account's page now shows user's home directory and default shell.
```
Comment (KKoukiou) [inline _posts/2023-03-22-cockpit-288.md:30]: account's detail page ?

File: _posts/2023-03-22-cockpit-288.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+![223722047-2885b1d2-2bbb-4694-8135-398d51a79387]({{ site.baseurl }}/images/288-accounts-show-shell-and-home-directory.png)
+
+## Services: Pinned units need to be re-done
```
Comment (KKoukiou) [inline _posts/2023-03-22-cockpit-288.md:34]: Maybe  add some 'Action needed' highlighter?

File: _posts/2023-03-22-cockpit-288.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+The data storage for "pinned" systemd units has changed. After upgrading to this Cockpit version, you need to re-pin units.
+
+## Allow specifying UID during account creation
```
Comment (KKoukiou) [inline _posts/2023-03-22-cockpit-288.md:38]: Accounts:

File: _posts/2023-03-22-cockpit-288.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+UID should be a integer bigger or same than 1000. There are use cases
```
Comment (KKoukiou) [inline _posts/2023-03-22-cockpit-288.md:40]: I think explaining the case with non-unique id is an overkill for release notes.

**Review by garrett — CHANGES_REQUESTED** [review]
I'm not quite done reviewing this, but I hope my suggestions go though.  We can apply these on the new PR, #617

**Pull request closed without merging**